test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb

```diff
@@ -283,8 +283,8 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
         end
         should "make outcome calculations" do
           assert_state_variable "total_hours", 45
-          assert_state_variable "minimum_hourly_rate", 6.08
-          assert_state_variable "total_hourly_rate", 6.07
+          assert_state_variable "minimum_hourly_rate", 6.5
+          assert_state_variable "total_hourly_rate", "6.12"
```

Comment:
It's not a fault of this commit, but I've noticed that this test doesn't seem to stub out current time. This means once we change the rates next year this test will pass at first, but when the new fiscal year starts, it will start failing. :watch: :bomb: